.github/workflows/ci_workflows.yml

```diff
@@ -48,6 +48,9 @@ jobs:
     needs: [initial_checks]
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
     with:
+      setenv: |
+        ARCH_ON_CI: $ARCH_ON_CI
```

Comment:
@ConorMacBride , this syntax didn't seem to do anything when I had it in the "envs" section. We'll see this time but do you know if this syntax is outright wrong?

Reply:
I think the issue is that `$ARCH_ON_CI` etc are being passed as a literal string and then evaluated inside the workflow where the variable is not defined. If you use the `env` context instead it should hopefully expand it when parsing the workflow file: `ARCH_ON_CI: ${{ env.ARCH_ON_CI }}` https://docs.github.com/en/actions/learn-github-actions/variables#using-the-env-context-to-access-environment-variable-values

Reply:
Nope. It is not working still.

![Screenshot 2023-03-14 170041](https://user-images.githubusercontent.com/2090236/225135223-ec5e1a9a-ca5e-4e77-bdef-87ff3bf4dcbf.jpg)

Reply:
It works when I don't use the OpenAstronomy workflow. For example, see the "32-bit and parallel" job from the same run.

![Screenshot 2023-03-14 170137](https://user-images.githubusercontent.com/2090236/225135476-53912305-4753-4c8c-879f-2d8801c25b51.jpg)

Reply:
Ah, my browser didn't show your second comment. Lemme try the `env.` trick.